ios/Sources/Repository/Authenticator.swift

```diff
@@ -0,0 +1,45 @@
+import DroidKaigiMPP
+import FirebaseAuth
+import Foundation
+
+class AuthenticatorImpl: Authenticator {
```

Comment:
Please make `public` 🙏🏼 

This Authenticator class will be placed in AppFeature's environment 🙏🏼

Reply:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
Do we also need a public initializer?

Reply:
I've added it 👍 cc47be5